schema/vrs/vrs-source.yaml

```diff
@@ -343,6 +343,17 @@ $defs:
         enum:
           - aa
           - na
+      sequence:
+        type:
+          $ref: "#/$defs/SequenceString"
+        description: >-
+          A :ref:`SequenceString` that is a literal representation of the referenced sequence.
+      moleculeType:
+        type: string
+        enum:
+          - genomic
```

Comment:
general discussion on genomic expressed concerns, possible alternate is "template"

Reply:
except "template" is a total non-starter. There are genomic variants that are not necessarily going to be involved as template sequence (e.g. variants in regulatory elements)

Reply:
Group consensus: we should cover the same concept types that RefSeq uses.

Reply:
lgtm